packages/jaeger-ui/src/utils/config/get-config.test.js

```diff
@@ -59,6 +59,7 @@ describe('getConfig()', () => {
     let getJaegerUiConfig;
 
     beforeEach(() => {
+      getConfig.apply({}, []);
```

Comment:
what does this do?

Reply:
This is reset memoization cache

https://github.com/alexreardon/memoize-one/issues/43

Reply:
thx. I wish they would've fixed that ticket by adding an explicit method like reset(), much better for readability.